planet/clients/orders.py

```diff
@@ -343,6 +343,8 @@ async def download_order(self,
             planet.exceptions.APIError: On API error.
             planet.exceptions.ClientError: If the order is not in a final
                 state.
+            planet.exceptions.ClientError: If more than one manifest file
+                per order.
```

Comment:
Unless this has ever been observed to happen I think it would be okay to omit this check for now. Getting no manifest or an incomplete manifest feels way more likely to me.

Reply:
Hmm yes that's what I was wondering too. @jreiberkyle thoughts? Should I just revert to what I had before or would you prefer to keep this in?

Reply:
Agreed those two are much more likely. In fact, the entire name of the file is `manifest.json` so instead of searching for a file that ends with `manifest.json` (which is a broader search and could possibly come up with multiple files), restrict the search to a file that is named `manifest.json`. This could be done pretty easily with a check for a match on `Path.name` ([ref](https://docs.python.org/3.7/library/pathlib.html#pathlib.PurePath.name)). And agreed that we would also want to wrap missing or corrupt `manifest.json` files in a `ClientError`